src/app/shared/modules/header/header.component.html

```diff
@@ -0,0 +1,24 @@
+<header>
```

Comment:
angular material has toolbar component. 
use it instead of your own implementation - https://material.angular.io/components/toolbar/overview